content/manuals/docker-hub/repos/manage/builds/setup.md

```diff
@@ -0,0 +1,301 @@
+Docker build system, add the [environment variable](index.md#environment-variables-for-builds)
+`DOCKER_BUILDKIT=0`. Refer to the [BuildKit](/manuals/build/buildkit/_index.md)
+page for more information on BuildKit.
+
+## Autobuild for teams
+
+When you create an automated build repository in your own user account, you
+can start, cancel, and retry builds, and edit and delete your own repositories.
+
+These same actions are also available for team repositories from Docker Hub if
+you are an owner. If you are a member of a
+team with `write` permissions you can start, cancel, and retry builds in your
+team's repositories, but you cannot edit the team repository settings or delete
+the team repositories. If your user account has `read` permission, or if you're
+a member of a team with `read` permission, you can view the build configuration
+including any testing settings.
+
+| Action/Permission     | Read | Write | Admin | Owner |
+| --------------------- | ---- | ----- | ----- | ----- |
+| view build details    |  x   |   x   |   x   |   x   |
+| start, cancel, retry  |      |   x   |   x   |   x   |
+| edit build settings   |      |       |   x   |   x   |
+| delete build          |      |       |       |   x   |
+
+### Service users for team autobuilds
+
+> [!NOTE]
+>
+> Only owners can set up automated builds for teams.
+
+When you set up automated builds for teams, you grant Docker Hub access to
+your source code repositories using OAuth tied to a specific user account. This
+means that Docker Hub has access to everything that the linked source provider
+account can access.
+
+For organizations and teams, it's recommended you create a dedicated service account to grant access to the source provider. This ensures that no
+builds break as individual users' access permissions change, and that an
+individual user's personal projects aren't exposed to an entire organization.
+
+This service account should have access to any repositories to be built,
+and must have administrative access to the source code repositories so it can
+manage deploy keys. If needed, you can limit this account to only a specific
+set of repositories required for a specific build.
+
+If you are building repositories with linked private submodules (private
+dependencies), you also need to add an override `SSH_PRIVATE` environment
+variable to automated builds associated with the account. For more information, see [Troubleshoot](troubleshoot.md#build-repositories-with-linked-private-submodules)
+
+1. Create a service user account on your source provider, and generate SSH keys for it.
+2. Create a "build" team in your organization.
+3. Ensure that the new "build" team has access to each repository and submodule you need to build.
+
+    Go to the repository's **Settings** page. On GitHub, add the new "build" team
+    to the list of **Collaborators and Teams**. On Bitbucket, add the "build" team
+    to the list of approved users on the **Access management** screen.
+
```

Comment:
it might help to make this section more readable by adding bullets under this number if that's possible:

3. Ensure that the new "build" team has access to each repository and submodule you need to build:
      - Go to the repository's **Settings** page. 
      - On GitHub, add the new "build" team to the list of **Collaborators and Teams**. 
      - On Bitbucket, add the "build" team to the list of approved users on the **Access management** screen.